connector_office_365/models/calendar_event.py

```diff
@@ -142,24 +207,24 @@ def office_365_open(self):
             'url': self.office_365_url
         }
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        records = super(CalendarEvent, self).create(vals_list)
```

Comment:
```suggestion
        records = super().create(vals_list)
```

no blocking this just python 3